src/commands/patchSite.js

```diff
@@ -0,0 +1,63 @@
+const fs = require('fs')
+const path = require('path')
+const tar = require('tar')
+const axios = require('axios')
+const chalk = require('chalk')
+const config = require('../config/config')
+
+async function loadGlobby () {
+  const globbyModule = await import('globby')
+  return globbyModule.globby || globbyModule.default || globbyModule
+}
+
+async function patchSite (folder, siteId) {
+  try {
+    if (!siteId) {
+      console.error(chalk.red('Site ID is required to patch a site.'))
+      return
+    }
+
+    console.log(chalk.blue('Patching site...'))
+
+    // Validate folder
+    if (!fs.existsSync(folder)) {
+      console.error(chalk.red(`Folder "${folder}" does not exist.`))
+      return
+    }
+
+    // Load globby dynamically
+    const globby = await loadGlobby()
+
+    // Create a tarball of the folder
+    const tarballPath = path.join(process.cwd(), 'site-patch.tar')
+    await tar.c(
+      {
+        gzip: true,
+        file: tarballPath,
+        cwd: folder
+      },
+      await globby(['**/*'], { cwd: folder })
+    )
+
+    // Read the tarball
+    const tarballData = fs.createReadStream(tarballPath)
+
+    // Send the tarball to the API
+    const response = await axios.patch(`${config.dpApiUrl}/sites/${siteId}`, tarballData, {
```

Comment:
Have you tested that this works? USually we use FormData along with the built in fetch API: https://github.com/hyphacoop/actions-distributed-press/blob/main/src/index.js#L54

Reply:
Yep! It worked previously:
![Screenshot 2024-11-14 at 12 01 38 AM](https://github.com/user-attachments/assets/aeeb1d50-3bde-41d3-a571-31e65f541125)

Just updated with this suggestion